src/css/style.css

```diff
@@ -1,5 +1,7 @@
 @charset "UTF-8";
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;700;800;900&display=swap');
+@import '@swarm-city/ui-library/dist/style.css';
```

Comment:
AFAIK most of the UI libraries don't need this and I wonder if we can get rid of tjis. Also I think we are not exporting this file.

Reply:
For the exporting part, see the issue linked in the comments.

But yeah, we should just be able to import the css file in the library's `index.ts` file. For some reason I just thought this was cleaner and potentially more flexible because it might make it possible to do some PostCSS shenanigans. No strong opinion / evidence though.

Reply:
I'll just keep it as is for now, we can always change it later.